src/utils/functions.ts

```diff
@@ -0,0 +1,7 @@
+export function getDiscountedPrice(price: number, discount: number): number {
+  return price - (price * discount);
+}
+
+export function priceFormatter(price: number): string {
+  return price.toLocaleString('en-US', { style: 'currency', currency: 'USD' });
+}
```

Comment:
Podrías documentar estas funciones utilizado jsdoc


```suggestion
/**
 * This function bla, bla, bla
 * @param {number} price - default Price 
 * @return {string} - formated price
 */
export function priceFormatter(price: number): string {
  return price.toLocaleString('en-US', { style: 'currency', currency: 'USD' });
}
```